internal/config/dns/webhook_dns.go

```diff
@@ -63,27 +61,42 @@ func (c *OperatorDNS) addAuthHeader(r *http.Request) error {
 	return nil
 }
 
-func (c *OperatorDNS) endpoint(bucket string, delete bool) (string, error) {
+func (c *WebhookDNS) endpoint(bucket string, delete bool) (string, error) {
 	u, err := url.Parse(c.Endpoint)
 	if err != nil {
 		return "", err
 	}
 	q := u.Query()
-	q.Add("bucket", bucket)
-	q.Add("delete", strconv.FormatBool(delete))
+	if bucket != "" {
+		q.Add("bucket", bucket)
+	}
+	if delete {
+		q.Add("delete", strconv.FormatBool(delete))
+	}
 	u.RawQuery = q.Encode()
 	return u.String(), nil
 }
 
-// Put - Adds DNS entries into operator webhook server
-func (c *OperatorDNS) Put(bucket string) error {
-	ctx, cancel := context.WithTimeout(context.Background(), defaultOperatorContextTimeout)
+// Put - Adds DNS entries into webhook webhook server
+func (c *WebhookDNS) Put(bucket string) error {
+	ctx, cancel := context.WithTimeout(context.Background(), defaultWebhookContextTimeout)
 	defer cancel()
-	e, err := c.endpoint(bucket, false)
+	ep, err := c.endpoint(bucket, false)
+	if err != nil {
+		return newError(bucket, err)
+	}
+	var srvRecords []SrvRecord
```

Comment:
This looks like a repetitive  action that can grow over time - maybe this is only need to be done in startup

Reply:
We need to capture `time.Now()` that present CreationDate for the bucket to the endpoint for retrieval later - not sure what you mean by startup?

Reply:
> We need to capture time.Now() that present CreationDate for the bucket to the endpoint for retrieval later - not sure what you mean by startup?

yes. I jut got a little bit confused about the exact spec there -

So here we create a srv record that points to minio domain name which should point to the load balancer ?